src/main.rs

```diff
@@ -4,6 +4,8 @@
 extern crate backtrace;
 #[macro_use(crate_version, crate_authors)]
 extern crate clap;
+extern crate libc;
+extern crate serde_json;
```

Comment:
`use ...` is the preferred way in Rust 2018.

Reply:
From what I see when running locally, this only works if we also have `edition = "2018"` in `Cargo.toml`.